src/react/features/connector/connectors.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Channel
+
+  Background:
+    Given I home
```

Comment:
Hmm? `Given I'm in home`?

Reply:
Should these be tested as features? I think these are not "user stories"... 🤔 

I propose deleting or completing this .

Reply:
Yes, that might be. I'll remove this for time being.